weekly-updates/Maria-Eriksson/Weekly-meetings.md

```diff
@@ -14,6 +15,58 @@
 * [7 September 2020](#date-7-september-2020)
 * [12 August 2020](#date-12-august-2020)
 
+
+### Date: 4 February 2021
+
+#### Who did you help this week?
+
+* BSc student about neuropsychology project
+* GA's collaborator (KW offered helpful advice - thank you!!)
+* Chat with neurosurgeon about neuropsychology
+* Partner
+  
+#### Who helped you this week?
+
+* Data buds chats and co-working
+
+#### What did you achieve?
+
+* Completed RSE course
+* Finished retrieving seizure and AED outcome for ALL PATIENTS OMG
+* Read commentary by EP & DJ
+* Worked on cross-validation script (but still stuck)
+* Had first official mentoring session with Isla, which was lovely
+* Follow-up re XNAT
+* Notes on data standards & open and reproduciblity practices in epilepsy surgery research - potential commentary paper
+
+#### What did you struggle with?
+
+* Month 2 of being in Finland, and am missing my London life (which I realise must sound bizarre considering the current situation) and my partner
+
+#### What would you like to work on next week?
+
+* Time to properly get project on GitHub (though through smaller steps, under a 'top level issue')
+	* Preliminary plan is to have 3 (private, for now) repositories: 1) paper on data standards & open and reproduciblity practices in epilepsy surgery research, 2) paper on "epilepsy surgery in children from 2000-2018: a single centre experience" (but need to get KW's thoughts on this and need to look into whether releasing the data in this paper is possible), and 3) predictive model scripts
+* Attend training session on 'Preparing for your postdoctoral career'
+* Add notes from mentoring session with Isla to GitHub
+
+#### Where do you need help from Kirstie?
+
+* Had a great (and very needed) chat about the current separation between my PhD and Turing time, and ways to bridge this separation.
+* Plan moving forward: 
+	* post 1-2 GitHub updates (issues, PRs) on WhitakerLab slack every day (help KW know more about what I'm doing)
+		* PRs can later be reviewed by someone else
+	* introduction on engageatturing slack
+	* inform KW about potential talks and conferences
+	* add weekly reflections to GitHub and share these with KW
```

Comment:
This looks great! Just a note to be clear that it isn't just to let me know more about what you're doing, its also to train the "open working" muscle - those skills to know how to identify specific tasks, marking progress on them (especially hard when they are hard to be totally "done"!) and articulating questions so that people can effectively help you.